python/ray/_private/structured_logging/utils.py

```diff
@@ -0,0 +1,31 @@
+from ray._private.structured_logging.formatters import TextFormatter
+from ray._private.structured_logging.filters import CoreContextFilter
+
+LOG_MODE_DICT = {
```

Comment:
nit: I think we can just define this in the constants.py?

Reply:
Moving to `constants.py` will cause a circular import.

* `LOG_MODE_DICT` requires `TextFormatter` and `CoreContextFilter` from `formatters.py` and `filters.py`.
* Both `filters.py` and `formatters.py` also import `constants.py`.

Reply:
Updated https://github.com/ray-project/ray/pull/45344/commits/2ec1b4f728e4a253611ff26142534ebe0a38b400